### ISSUE TITLE
Users list contains self

### ISSUE THREAD

**Issue by tszkanlo** [skygear-demo/swift-chat-uikit-demo#25]

**Comment by carmenlau** [comment]
In most of the app like slack, telegram, whatsapp. User can send message to himself, just a place to store message. I think we can keep self in the user list, just need to know sure clicking it can create one man conversation and save message there. What do you think? :)

**Comment by zacharylo** [comment]
Agree, but for the user list, i think the "me" user shd be at the top of the list (this may regard as demo app issue)
